src/implementations/twig/components/button/button.html.twig

```diff
@@ -50,6 +54,12 @@
   {% endfor %}
 {% endif %}
 
+{% if icon.name is defined and icon.name is not empty %}
+  {% set _icons = [icon] %}
+{% elseif icon[1] is defined and icon[1] is not empty %}
```

Comment:
If I understand correctly, the button can now have an array of icons. This is worth mentioning in the doc and in the data model commented

Reply:
yep, it's the same we do in the link template. I think it would not be too late, anyway, to actually move the extra_classes always inside the icon named object, or even better removing completely this object